language/move-analyzer/editors/code/src/commands/lsp_command.ts

```diff
@@ -22,3 +22,21 @@ export async function textDocumentDocumentSymbol(
     // Send the request to the language client.
     return client.sendRequest(DocumentSymbolRequest.type, params);
 }
+
+
+/**
+ * An LSP command textDocument/hover
+ */
+export async function textDocumentHover(
```

Comment:
Just for my own information - this is only used in tests, right (and would not be needed otherwise)?

Reply:
Yes - this is only used in the integration tests. I've followed the same format as the existing LSP command in that file (`textDocumentDocumentSymbol`).